tripleo-quickstart-config/roles/libvirt/setup/overcloud/templates/ironic_nodes.json.j2

```diff
@@ -0,0 +1,39 @@
+{% set lvars = { 'host_ip' : '192.168.122.1', 'pxe_network' : False} %}
+{% for network in networks %}
+{% if (not (network.forward_mode is defined and network.forward_mode == 'nat') and lvars['pxe_network'] == False) %}
+{% if lvars.update({'pxe_network' : network.name}) %}{% endif %}
+{% endif %}
+{% if network.address is defined and lvars['host_ip'] == '192.168.122.1' %}
+{% if lvars.update({'host_ip' : network.address}) %}{% endif %}
+{% endif %}
+{% endfor %}
+{
+  "nodes": [
+  {% for node in overcloud_nodes %}
+    {
+      "name": "{{ node.name|replace('_', '-') }}",
+      "driver": "ipmi",
+      "resource_class": "baremetal",
+      "driver_info": {
+        "ipmi_user": "admin",
+        "ipmi_password": "password",
+        "ipmi_addr": "192.168.122.1",
+        "ipmi_port": "{{ node.virtualbmc_port }}",
+        "deploy_kernel": "http://172.22.0.1/images/tinyipa-stable-rocky.vmlinuz",
```

Comment:
We may parametrize the release (s/stable-rocky/.../) eventually

Reply:
Ack yeah this was a cut/paste from the readme but we can add a variable when needed